src/engine/txn_engine_helper.h

```diff
@@ -227,6 +229,32 @@ class TxnEngineHelper {
 
   static void RegularUpdateSafePointTsHandler(void *arg);
   static void RegularDoGcHandler(void *arg);
+
+  static void AsyncCommitTimestamps(int64_t region_id, std::string key, int64_t start_ts, int64_t for_update_ts,
+                                    int64_t lock_min_commit_ts, int64_t max_commit_ts, int64_t &final_min_commit_ts);
+
+  static butil::Status WriteLock(
```

Comment:
Change this function name to a more understandable name, like "GenPrewriteDataAndLock"